common/lib/capa/capa/tests/test_responsetypes.py

```diff
@@ -1106,6 +1245,96 @@ def test_checkbox_group_grade(self):
         # No choice 3 exists --> mark incorrect
         self.assert_grade(problem, 'choice_3', 'incorrect')
 
+    def test_checkbox_group_valid_grading_schemes(self):
+        # Checkbox-type problems only allow one partial credit scheme.
+        # Change this test if that changes.
+        problem = self.build_problem(choice_type='checkbox',
+                                     choices=[False, False, True, True],
+                                     credit_type='edc,halves,bongo')
+        with self.assertRaises(LoncapaProblemError):
+            input_dict = {'1_2_1': 'choice_1'}
+            problem.grade_answers(input_dict)
+
+        # 'bongo' is not a valid grading scheme.
+        problem = self.build_problem(choice_type='checkbox',
+                                     choices=[False, False, True, True],
+                                     credit_type='bongo')
```

Comment:
Multiline breaks should be like this:
```python
problem = self.build_problem(
    choice_type='checkbox',
    choices=[False, False, True, True],
    credit_type='bongo'
)
```

See https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines